### DISCUSSION TITLE
yetiforce and google calendar

### DISCUSSION THREAD

**Discussion by Gazzxy** [YetiForceCompany/YetiForceCRM#14677] [How to]
is it possible to connect this things calandar to google ? google calendar seems to take the caldav link then does nothing with it. just wana check its even possible before I waste too much more time on it

**Comment by z0lo13** [comment]
@Gazzxy 
You will need to develop the integration with google calendar on your own.
You need to create the CRON job which will synchronize the table data with the Calendar API.

https://developers.google.com/calendar/quickstart/php

**Comment by Gazzxy** [comment]
sounds like fun and a backburner sorta problem not really my ballpark heh

**Reply by mariuszkrzaczkowski** [reply]
you can order integration https://yetiforce.com/en/marketplace/development-support

**Reply by rioshaz** [reply]
@mariuszkrzaczkowski any idea how much will it cost ...

**Reply by mariuszkrzaczkowski** [reply]
hard to say, it required to read the structure of the google api calendar carefully

20-40 hours, I suppose
